vizro-core/changelog.d/20240305_142454_huong_li_nguyen_fix_card_validation.md

```diff
@@ -0,0 +1,46 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Highlights ✨
+
+- A bullet item for the Highlights ✨ category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+<!--
+### Removed
+
+- A bullet item for the Removed category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+<!--
+### Added
+
+- A bullet item for the Added category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+
+### Changed
+
+- Allow `Page.title` to be inserted directly into `Card.href` as a relative link. ([#347](https://github.com/mckinsey/vizro/pull/347))
```

Comment:
```suggestion
- Allow `Page.title` to be inserted directly into `Card.href` as a relative link without the need to provide a `/`. ([#347](https://github.com/mckinsey/vizro/pull/347))
```

Then I think it's super clear :) I first read the sentence as: I insert the title as a relative link so I need to add a `/`

Reply:
Or we just shorten to: Allow `Page.title` to be inserted directly into `Card.href` ?

Because it's not only the slash that prevented that from doing that before